common/util.go

```diff
@@ -387,6 +387,13 @@ func IsNotFoundError(err error) bool {
 	return errors.As(err, &notFoundErr)
 }
 
+func ErrorHash(err error) string {
+	if err != nil {
+		return fmt.Sprintf("0x%x", farm.Fingerprint32([]byte(err.Error())))
+	}
+	return "0x0000000"
```

Comment:
wrong number of zeros
```suggestion
	return "00000000"
```
(or 0x... if you really like the prefix)

Reply:
I just want it to be consistant - for other values it starts with 0x.